### PR TITLE
Added Chapter 11

### PR DESCRIPTION
**Pull request by MarkWignall**
Added Chapter 11 Answers

### CODE REVIEW

File: chapter11/mdw/Ex2.pl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+use strict;
+use warnings;
+
+use v5.12;
+
+require 'Island.pm';
+
+my($sec, $min, $hour, $mday, $mon, $year, $wday) = localtime; 
+
+my $day = Island::number_to_day_name( $wday -1 );
```
Comment (thatguy59) [inline chapter11/mdw/Ex2.pl]: Breaks on Sundays?

From the localtime documentation:
> `$wday` is the day of the week, with 0 indicating Sunday and 3 indicating Wednesday

File: chapter11/mdw/Island.pm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+package Island 0.11;
+
+#Translations
+our @day = qw(ark dip wap sen pop sep kir);
+our @month = qw(diz pod bod rod sip wax lin sen kun fiz nap dep);
+
+#This method turns the number of the day into its name
+sub number_to_day_name { 
+	my ( $num ) = @_;
+	if ( $num && ( $num >= 0 ) && ( $num < 7 ) ){
```
Comment (thatguy59) [inline chapter11/mdw/Island.pm]: What about non-numerics?

Comment (thatguy59) [inline chapter11/mdw/Island.pm]: Also, `if ( $num )` will be false for `0` right? So this also breaks on Sundays.

Comment (MarkWignall) [inline chapter11/mdw/Island.pm]: Yeah, I forgot about non-numerics. See previous comments re: Sundays.

**Pull request merged**